src/EventListener/RemoveSessionSubscriber.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace eLife\Journal\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+final class RemoveSessionSubscriber implements EventSubscriberInterface
```

Comment:
IIUC this removes the session in certain cases (the session being existing and invalid?), rather than every time, so could have a more precise name unless that decision is in the DI configuration